cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -275,6 +275,76 @@ These metrics are available in a stream context:
    * - ``system.collect.bytes``
      - Number of bytes collected by the stream
 
+These metrics on request and response are available for system and user services:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 60 40
+
+   * - Request and Response Metric
+     - Description
+   * - ``system.request.received``
+     - Number of requests received for the service
+   * - ``system.response.successful``
+     - Number of successful responses sent
+   * - ``system.response.{server-error, client-error}``
+     - Number of ``server-error`` or ``client-error`` responses sent
+
+These metrics on logging are available on the application context:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 60 40
+
+   * - Application Logging Metric
+     - Description
+   * - ``system.app.log.{error, info, warn}``
+     - Number of ``error``, ``info``, or ``warn`` log messages logged by the application
+
+These metrics on logging are available for the system services, in the system component context:
```

Comment:
`These metrics are available for the system services, in the system component context:`